### PR TITLE
TCP server-client

### PR DESCRIPTION
**Pull request by NeverovDS**

### CODE REVIEW

File: TCP-client.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"net"
+	"strings"
+)
+import "fmt"
+import "bufio"
+import "os"
```
Comment (burov) [inline TCP-client.go]: Please specify all imports in one section in sorted way

File: TCP-client.go [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+func main() {
+
```
Comment (burov) [inline TCP-client.go]: Redundant blank line

File: TCP-client.go [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+	conn, err := net.Dial("tcp", ":8081")
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
```
Comment (burov) [inline TCP-client.go]: It's better to add blank line after this bracket to separate initialisation from for loop

File: TCP-client.go [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+		reader := bufio.NewReader(os.Stdin)
+		fmt.Print("Text to send: ")
+		text, err := reader.ReadString('\n')
+		if err != nil {
+			fmt.Println(err)
+			return
+		}
+		if strings.TrimSpace(string(text)) == "exit" {
+			fmt.Println("client usnul")
```
Comment (burov) [inline TCP-client.go]: oh my universe, ```usnul``` - what is mean ?

File: TCP-client.go [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+			return
+		} else {
```
Comment (burov) [inline TCP-client.go]: Else not needed if ``` strings.TrimSpace(string(text)) == "exit"``` you will return from function

File: TCP-srever.go [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+		}
+		fmt.Print("Message Received:", string(message))
+		// Процесс выборки для полученной строки
+		message = strings.TrimSpace(string(message))
+		r, err1 := regexp.MatchString(`^[0-9]+$`, message)
+		if err1 != nil {
+			fmt.Println(err1)
+			return
+		}
+		if r == true {
```
Comment (burov) [inline TCP-srever.go]: ```if r {``` ?

File: TCP-srever.go [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+			newMessage, _ := strconv.Atoi(message)
+			newMessage *= 2
+			conn.Write([]byte(strconv.Itoa(newMessage) + "\n"))
+		}
+		if r == false {
```
Comment (burov) [inline TCP-srever.go]: If return from function in case of r == true, you don't need to check ```r == false```

File: TCP-client.go (hunk 4 above)
Comment (wshaman) [reply TCP-client.go]: :smiley:

**Pull request closed without merging**